### ISSUE TITLE
Double quotes in title of person tiddler cause unwanted views

### ISSUE THREAD

**Issue by clsturgeon** [clsturgeon/MemoryKeeper#4]
ViewTemplates are not supporting double quotes in the title of person tiddler.  Fixed in 0.0.05.

However, there are larger issues. Javascript crashes in TiddlyMap widgets. This is not likely an issue with TiddlyMap, rather an issue on how it is used.  You may experience a situation that is not recoverable.

Some testing has not found issues with double quotes in the event, source, or note tiddler titles. Double quotes in the place tiddler however is an issue, specifically when the mapit tag is added to the place tiddler.

Bottom line: At this time ensure no tiddler title contains double-quotes.
